scripts/ct/run.sh

```diff
@@ -98,7 +98,7 @@ TTY=''
 if [[ -t 1 ]]; then
     TTY='-t'
 fi
-docker exec -i $TTY "$ERLANG_CONTAINER" bash -c 'git config --global --add safe.directory /emqx'
+docker exec -i $TTY "$ERLANG_CONTAINER" bash -c 'git config --global --add safe.directory "*"'
```

Comment:
Just confirming: is this change intended here?  It might have different behaviors depending on the work dir.

Reply:
+1 to Thales, what is the reason for this change?

Reply:
Without this change, some actions won't work successfully (authn/authz), and it will report some repos are unsafe (like quic), even if I add these repos one by one, these actions still won't succeed.

Reply:
@lafirest another option is to use the same user and group id when building inside the container
Haven't looked what are corresponding compose options, but this one worked for me (running make inside emqx-builder):
```
-v ~/.cache:/.cache -u $(id -u $USER):$(id -g $USER)
```